crossbeam-skiplist/src/lib.rs

```diff
@@ -257,3 +258,6 @@ pub mod set;
 #[cfg(feature = "std")]
 #[doc(inline)]
 pub use crate::{map::SkipMap, set::SkipSet};
+
+/// Traits for key comparison in maps.
```

Comment:
It would be better to move the documentation to the equivalent.rs file side as with other modules.